config/routes.rb

```diff
@@ -2,6 +2,7 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
+  mount Messages::API => '/'
```

Comment:
I think we should mount the API under the `/api` path. That way we don't have to repeat that on every endpoint we build

```suggestion
  mount Messages::API => '/api'
```

Reply:
Done!